### PR TITLE
[TACHYON-986] Increase usability of TachyonFileSystem

### PR DESCRIPTION
**Pull request by calvinjia**
https://tachyon.atlassian.net/browse/TACHYON-986

### CODE REVIEW

**Comment by yupeng9** [comment]
LGTM

**Comment by danielfree** [comment]
How about createDependency() method?

File: clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystem.java [hunk 1]
```diff
@@ -63,9 +63,48 @@ public long create(TachyonURI path, long blockSize, boolean recursive) {
...
    *
-   * @param file the TachyonFile to delete recursively
+   * @param path the Tachyon path of the file
+   * @return the fileId of the created file
+   * @throws InvalidPathException if the provided path is invalid
+   * @throws FileAlreadyExistException if the file being written to already exists
+   * @throws BlockInfoException if the provided block size is invalid
+   * @throws IOException if the master cannot create the file.
+   */
+  public long createEmptyFile(TachyonURI path) throws IOException, InvalidPathException,
```
Comment (gpang) [inline clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystem.java]: Can the implementation for this method just call the other createEmptyFile(path, options)?

**Comment by gpang** [comment]
I just had a minor comment. Thanks!

**Comment by calvinjia** [comment]
@danielfree Since there are no current users of that method (its purely client facing, not used internally), I would prefer to include it when the lineage client is implemented.

File: clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystem.java (hunk 1 above)
Comment (calvinjia) [inline clients/unshaded/src/main/java/tachyon/client/file/TachyonFileSystem.java]: Good point, thanks.

**Comment by calvinjia** [comment]
Jenkins, test this please.

**Comment by chen-tian** [comment]
LGTM

**Comment by gpang** [comment]
LGTM

**Pull request merged**